core/src/main/java/org/apache/calcite/sql/validate/SqlValidator.java

```diff
@@ -931,6 +931,18 @@ boolean validateModality(SqlSelect select, SqlModality modality,
      */
     Config withCallRewrite(boolean rewrite);
 
+    /**
+     * Returns whether to allow unresolved types during validation.
+     */
+    @ImmutableBeans.Property
+    @ImmutableBeans.BooleanDefault(false)
+    boolean handleUnknownTables();
```

Comment:
nit: /handleUnknownTables/allowUnknownTables/ ?

Reply:
Sure, I can change it to allowUnknownTables